README.md

```diff
@@ -21,7 +21,7 @@ It is using [podman go bindings](https://github.com/containers/podman/tree/main/
 
 ## Installation
 
-Building from source (Linux, Windows) or installing packaged versions are detailed in [install guide](install.md).
+Building from source (Linux, Windows and Darwin) or installing packaged versions are detailed in [install guide](install.md).
```

Comment:
You sure you want to call this Darwin, or just Mac?

Reply:
You are right, I think it is better to call it Mac, will update the PR and change it to Mac.